documentation/modules/new-migrating-virtual-machines-cli.adoc

```diff
@@ -0,0 +1,1046 @@
+        type: multus
+      source:
+        id: <source_network_id>
+  provider:
+    source:
+      name: <source_provider>
+      namespace: <namespace>
+    destination:
+      name: <destination_provider>
+      namespace: <namespace>
+EOF
+----
+<1> Allowed values are `pod` and `multus`.
+<2> Specify a network attachment definition for each additional {virt} network.
+<3> Required only when `type` is `multus`. Specify the namespace of the {virt} network attachment definition.
+endif::[]
+
+ifdef::ostack[]
+[start=3]
+. Create a `NetworkMap` manifest to map the source and destination networks:
++
+[source,yaml,subs="attributes+"]
+----
+$  cat << EOF | {oc} apply -f -
+apiVersion: forklift.konveyor.io/v1beta1
+kind: NetworkMap
+metadata:
+  name: <network_map>
+  namespace: <namespace>
+spec:
+  map:
+    - destination:
+        name: <network_name>
+        type: pod <1>
+      source:<2>
+        id: <source_network_id>
+        name: <source_network_name>
+    - destination:
+        name: <network_attachment_definition> <3>
+        namespace: <network_attachment_definition_namespace> <4>
+        type: multus
+      source:<5>
+        id: <source_network_id>
+        name: <source_network_name>
+  provider:
+    source:
+      name: <source_provider>
+      namespace: <namespace>
+    destination:
+      name: <destination_provider>
+      namespace: <namespace>
+EOF
+----
+<1> Allowed values are `pod` and `multus`.
+<2> You can use either the `id` _or_ the `name` parameter to specify the source network. For `id`, specify the {osp} network UUID.
+<3> Specify a network attachment definition for each additional {virt} network.
+<4> Required only when `type` is `multus`. Specify the namespace of the {virt} network attachment definition.
+<5> Same as `source` above.
+endif::[]
+
+ifdef::cnv[]
+[start=3]
+. Create a `NetworkMap` manifest to map the source and destination networks:
++
+[source,yaml,subs="attributes+"]
+----
+$  cat << EOF | {oc} apply -f -
+apiVersion: forklift.konveyor.io/v1beta1
+kind: NetworkMap
+metadata:
+  name: <network_map>
+  namespace: <namespace>
+spec:
+  map:
+    - destination:
+        name: <network_name>
+        type: pod <1>
+      source: <2>
+        name: <network_name>
+        type: pod
+    - destination:
+        name: <network_attachment_definition> <3>
+        namespace: <network_attachment_definition_namespace> <4>
+        type: multus
+      source:
+        name: <network_attachment_definition> <5>
+        namespace: <network_attachment_definition_namespace> <6>
+        type: multus
+  provider:
+    source:
+      name: <source_provider>
+      namespace: <namespace>
+    destination:
+      name: <destination_provider>
+      namespace: <namespace>
+EOF
+----
+<1> Allowed values are `pod` and `multus`.
+<2> You can use either the `id` _or_ the `name` parameter to specify the source network. For `id`, specify a pod network that is similar to the destination.
```

Comment:
`<2>` is incorrect for OpenShift provider (there's no `id`), I think we can drop `<2>` from here